pkg/crc/network/utils.go

```diff
@@ -92,3 +94,24 @@ func UriStringForDisplay(uri string) (string, error) {
 	}
 	return uri, nil
 }
+
+func CheckCRCLocalDNSReachableFromHost(clusterName, domainName, appsDomainName string) error {
+	ip, err := net.LookupIP(fmt.Sprintf("api.%s.%s", clusterName, domainName))
+	if err != nil {
+		return err
+	}
+	logging.Debugf("api.%s.%s resolved to %s", clusterName, domainName, ip)
+
+	if runtime.GOOS != "darwin" {
```

Comment:
👍

Reply:
I know this is needed, but less charmed about it...

Reply:
We could go add some `utils_darwin.go` , `utils_linux.go`... to deal with the os-specific bits, but this seemed overkill.

Reply:
I was considering the same... and agree about the overkill. just concerned this gets forgotten.